realm/realm-jni/src/io_realm_internal_LinkView.cpp

```diff
@@ -176,3 +176,18 @@ JNIEXPORT jboolean JNICALL Java_io_realm_internal_LinkView_nativeIsAttached
     } CATCH_STD()
     return 0;
 }
+
+JNIEXPORT jboolean JNICALL Java_io_realm_internal_LinkView_nativeFind
+  (JNIEnv *env, jobject, jlong nativeLinkViewPtr, jlong rowIndex)
+{
+    TR_ENTER_PTR(nativeLinkViewPtr)
+    try {
+        jlong ndx = LV(nativeLinkViewPtr)->find(rowIndex);
+        if (ndx == to_jlong_or_not_found(realm::not_found)) {
```

Comment:
Yes, something like `return to_jlong_or_not_found(LV(nativeLinkViewPtr)->find(S(rowIndex)));` Casting `rowIndex` to a `size_t` can be done by the `S()` macro.